library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -134,6 +138,10 @@ public int getFrameCount() {
     return state.frameLoader.getFrameCount();
   }
 
+  public int getFrameIndex() {
```

Comment:
Add a javadoc for this method (returns the current frame index in range 0 to count-1, or -1 if no frame is displayed.